downloads/swan-lake-release-notes/swan-lake-alpha4/swan-lake-alpha4-release-note.md

```diff
@@ -310,21 +334,18 @@ rabbitmq:Client client = check new(”localhost”, 9090);
 ```
 
 ##### Security Updates
+
  - Removed encrypted passwords and hashed passwords support for Basic Auth file user store authentication.
```

Comment:
This is not clear.

Reply:
@chanakaudaya

Reply:
There was a support to add encrypted or hashed passwords for Basic Auth file user store authentication. We remove that support with the update done for `configurable` feature.

Reply:
Resolving since the idea is clear.